lib/src/model/weightranges_json.dart

```diff
@@ -0,0 +1,29 @@
+/// lower : "23"
+/// upper : "23"
+/// weight : "1"
+
+class WeightRanges {
```

Comment:
We can use the actual chain files. Rather than our interpretation.

Reply:
we could. i think it would be better if we ditched a lot of these weird intermediary types outright, tbh. but these are the data model types the wallet uses right now, and i want to get this out of the way asap, so we have to be able to pass them to client apps.